src/m1_facilitator/Config/Config.ts

```diff
@@ -0,0 +1,189 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import fs from 'fs-extra';
+import yaml from 'js-yaml';
+
+/**
+ * The class holds database configurations.
+ */
+export class DBConfig {
+  /** Database path */
+  public path: string;
+
+  public constructor() {
+    this.path = '';
+  }
+}
+
+/**
+ * It holds chain information like .
+ */
+export class Chain {
+  /** Chain rpc endpoint. */
+  public readonly nodeRpcEndpoint: string;
+
+  /** Subgraph web socket endpoint */
+  public readonly subGraphWsEndpoint: string;
+
+  /** Subgraph rpc endpoint */
+  public readonly subGraphRpcEndpoint: string;
+
+  /** Worker address. */
+  public readonly worker: string;
+
+  /** Worker password. */
+  private readonly _password?: string;
+
+  public constructor(
+    nodeRpcEndpoint: string,
+    subGraphWsEndpoint: string,
+    subGraphRpcEndpoint: string,
+    worker: string,
+    password?: string,
+  ) {
+    this.nodeRpcEndpoint = nodeRpcEndpoint;
+    this.subGraphWsEndpoint = subGraphWsEndpoint;
+    this.subGraphRpcEndpoint = subGraphRpcEndpoint;
+    this.worker = worker;
+    this._password = password;
+  }
+
+  /**
+   * Get the password for unlocking worker.
+   */
+  public get password(): string | undefined {
```

Comment:
Do we need to return password? We can just add unlocked account to web3 wallet. This getter can be consumed in wrong way.

Reply:
It's removed for now. At time of service implementation we will better know the unlock behaviour.